po/zh-CN.po

```diff
@@ -12751,10 +12751,26 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"```rust,editable\n"
```

Comment:
It's fine to decide not to translate the code — if you think this would make sense in general, then we can add a special `<!-- mdbook-xgettext: skip -->` comment right before the code block. That will prevent is from being extracted into the `messages.pot` file the next time someone refreshes the translation. See #1327 and related issues for details.

Reply:
I'll be submitting a patch to fix all this later, but for now, if I refresh the translation, it's going to break my other PR's!